test/checkbox-test.js

```diff
@@ -1,315 +1,33 @@
-		} );
-
-		$input.click();
-		assert.equal( triggered, true, "changed event triggered" );
-
-		$element.remove();
-	} );
-
-	QUnit.test( "should trigger changed event when clicking on input element", function( assert ) {
-		var $element = $( html ).find( "#CheckboxUncheckedEnabled" ).clone();
-		var $input = $element.find( 'input[type="checkbox"]' );
-		$element.appendTo( document.body ); // Append to body to capture clicks
-
-		// Initialize checkbox
-		var $chk = $element.find( "label" ).checkbox();
-
-		var triggered = false;
-		$element.on( "changed.fu.checkbox", function() {
-			triggered = true;
-		} );
-
-		$input.click();
-		assert.equal( triggered, true, "changed event triggered" );
-
-		$element.remove();
-	} );
-
-	QUnit.test( "should toggle checkbox container visibility", function( assert ) {
-		var $element = $( html ).find( "#CheckboxToggle" ).clone();
-		var $container = $element.find( ".checkboxToggle" );
-		$element.appendTo( document.body ); // Append to body to check visibility
-
-		// Initialize checkbox
-		var $chk = $element.find( "label" ).checkbox();
-
-		assert.equal( $container.is( ":visible" ), false, "toggle container hidden by default" );
-		$chk.checkbox( "check" );
-		assert.equal( $container.is( ":visible" ), true, "toggle container visible after check" );
-		$chk.checkbox( "uncheck" );
-		assert.equal( $container.is( ":visible" ), false, "toggle container hidden after uncheck" );
-
-		$element.remove();
-	} );
-
-	QUnit.test( "should destroy checkbox", function( assert ) {
-		var $element = $( html ).find( "#CheckboxCheckedEnabled" ).clone();
-
-		// Initialize checkbox
-		var $chk = $element.find( "label" ).checkbox();
-		var originalMarkup = $element.find( "label" )[ 0 ].outerHTML;
-
-		assert.equal( $element.find( "#Checkbox1" ).length, 1, "checkbox exists in DOM by default" );
-
-		var markup = $chk.checkbox( "destroy" );
-
-		assert.equal( originalMarkup, markup, "returned original markup" );
-		assert.equal( $element.find( "#Checkbox1" ).length, 0, "checkbox removed from DOM" );
-	} );
-} );
+	var $ = require('jquery');
+
+	require('bootstrap');
+	require('fuelux/checkbox');
+
+
+	QUnit.module('Fuel UX Checkbox', {
+		beforeEach: require('./checkbox-tests/helpers').setup
+	}, function runCheckboxTests () {
+		QUnit.test('should be defined on jquery object', function isCheckboxDefined (assert) {
+			assert.ok($().checkbox, 'checkbox method is defined');
+		});
+
+		QUnit.test('should return element', function doesCheckboxReturnCheckbox (assert) {
+			assert.ok(this.$checkedEnabled.checkbox() === this.$checkedEnabled, 'checkbox should be initialized');
+		});
+
+		require('./checkbox-tests/initialization')(QUnit);
+		require('./checkbox-tests/disable')(QUnit);
+		require('./checkbox-tests/checked')(QUnit);
+		require('./checkbox-tests/return')(QUnit);
+		require('./checkbox-tests/get-value')(QUnit);
+		require('./checkbox-tests/events')(QUnit);
+		require('./checkbox-tests/visibility')(QUnit);
```

Comment:
Looking closer at this, checkbox visibility tests are fully defined. Not sure where you are seeing it is skipped? Tree is the only thing that has skipped tests (and those were added by a different PR).